scripts/bert/baidu_ernie_data.py

```diff
@@ -0,0 +1,94 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=line-too-long
+"""GLUEBenchmark corpora."""
+
+__all__ = ['BaiduErnieXNLI']
+
+import zipfile
+import os
+
+from mxnet.gluon.utils import download, check_sha1, _get_repo_file_url
+
+from gluonnlp.data.dataset import TSVDataset
+from gluonnlp.data.registry import register
+from gluonnlp.base import get_home_dir
+
+_glue_s3_uri = 's3://apache-mxnet/gluon/dataset/Glue/'
+
+class _BaiduErnieDataset(TSVDataset):
+    def __init__(self, root, segment, **kwargs):
+        root = os.path.expanduser(root)
+        if not os.path.isdir(root):
+            os.makedirs(root)
+        self._root = root
+        filename = os.path.join(self._root, '%s.tsv' % segment)
+        super(_BaiduErnieDataset, self).__init__(filename, **kwargs)
```

Comment:
can we use zipFile API and download and extract from `https://ernie.bj.bcebos.com/task_data.tgz` directly?

Reply:
Yes. I used urlretrieve and tarfile to do these things.